libscript/src/script-execute.cpp

```
@@ -202,10 +209,32 @@ class MCScriptForeignInvocation
             break;
         
         case kMCScriptForeignHandlerLanguageC:
```

We should probably throw an error if a variadic foreign handler binds to java since it's not yet supported

---

I fixed this in the MCJavaCheckSignature - it throws a binding error if there is a variadic parameter present.